metricbeat/docs/fields.asciidoc

```diff
@@ -30122,6 +30123,317 @@ format: duration
 
 --
 
+[[exported-fields-tomcat]]
+== Tomcat fields
+
+Tomcat module
+
+
+
+
+[float]
+=== cache
+
+Catalina Cache metrics from the WebResourceRoot
+
+
+*`tomcat.cache.mbean`*::
++
+--
+Mbean that this event is related to
+
+type: keyword
+
+--
+
+*`tomcat.cache.hit.total`*::
++
+--
+The number of requests for resources that were served from the cache
+
+type: long
+
+--
+
+*`tomcat.cache.size.total.kb`*::
++
+--
+The current estimate of the cache size in kB
```

Comment:
```suggestion
The current estimate of the cache size in kilobytes
```

A tiny thing (and elsewhere too). KB is usually only used if it follows a number, otherwise it's usually spelled out in full (where "usually" means "as defined in the Microsoft manual of Style").

Reply:
That's really interesting! I didn't know it. Fixed!